README.md

```diff
@@ -161,6 +171,25 @@ val StoreBuffering: LitmusTest<*> = litmusTest({
 }
 ```
 
+There also is an alternative, even shorter syntax based on infix functions:
```

Comment:
Let's remove anything non-JCStress related from the Readme in this PR

Reply:
To keep things easier, I removed all changes to README, we'll apply them once all recent PR-s are merged into `developent`